include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -122,6 +122,12 @@ class MGTwoLevelTransfer
    */
   void
   restrict_and_add(VectorType &dst, const VectorType &src) const;
+
+  /**
+   * Perform interpolation.
```

Comment:
I think we should be more specific to guide the users with less experience regarding the difference.
```suggestion
   * Perform interpolation of a solution vector from the fine level to the coarse level.
   * This function is different from restriction, where a weighted residual is
   * transferred to a coarser level (transposition of prolongation matrix).
```